.dockerignore

```diff
@@ -37,3 +37,5 @@ my_*
 **/*.gz
 **/*.bz2
 **/*.png
+**/*.sif
+*.sif
```

Comment:
```suggestion
**/*.sif
```

Reply:
@franchuterivera, for some reason, I'm not able to commit the suggestion ("no changes were made to the code" !!!).
Please try on your side, I'll merge afterwards.